CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 42.0.0
```

Comment:
I think something's gone wrong with this commit - it shouldn't be in here, it's already in `main`. Try a rebase first, and if that doesn't remove it probably just delete this commit from this PR.

Reply:
Strange! I'm not sure how this got in here, perhaps I mistakenly included it by going too far back in the history during an interactive rebase. I've rebased against main again and it's disappeared now.